gpumon/gpumon_client.ml

```diff
@@ -17,14 +17,13 @@ open Xcp_client
 
 let xml_url () = "file:" ^ xml_path
 
-module Client = Gpumon_interface.Client(struct
```

Comment:
With this gone, do we still need ` open Gpumon_interface`?

Reply:
The rpc function uses a lot of Gpumon_interface values - ones that are actually common to most daemon interfaces. I've wondered about factoring out all this common code into a functor, but I'm not sure how convenient it would be for any Xapi code referring to different interfaces, since it would just make daemon interfaces shorter at the expense of making references more verbose.